searx/data/__init__.py

```diff
@@ -3,7 +3,7 @@
 
 
 __init__ = ['ENGINES_LANGUGAGES', 'CURRENCIES', 'USER_AGENTS', 'EXTERNAL_URLS', 'WIKIDATA_UNITS', 'EXTERNAL_BANGS',
-            'bangs_loader', 'ahmia_blacklist_loader']
+            'OSM_KEYS_TAGS', 'bangs_loader', 'ahmia_blacklist_loader']
```

Comment:
@dalf do you know where this `__init__` variable is used?  I have problems to find the usage.

Reply:
It should `__all__ ` not `__init__` !

Reply:
updated